st2common/st2common/models/api/keyvalue.py

```diff
@@ -114,17 +120,22 @@ def from_model(cls, model, mask_secrets=True):
         if 'id' in doc:
             del doc['id']
 
-        if model.expire_timestamp:
+        if getattr(model, 'expire_timestamp', None) and model.expire_timestamp:
             doc['expire_timestamp'] = isotime.format(model.expire_timestamp, offset=False)
 
         encrypted = False
-        if model.secret:
+        secret = getattr(model, 'secret', False)
+        if secret:
             encrypted = True
 
-        if not mask_secrets and model.secret:
+        if not mask_secrets and secret:
             doc['value'] = symmetric_decrypt(KeyValuePairAPI.crypto_key, model.value)
             encrypted = False
 
+        scope = getattr(model, 'scope', SYSTEM_KV_PREFIX)
```

Comment:
I like "user" and "global" scopes as well. I even anticipate more scopes like "environment" etc. Our current story for key value store is everything is global and you can "refer" to a variable via  {{system.foo}} even though "foo" is the actual key name. So for backward compatibility, I am setting the scope to "system" and don't want to introduce another name "global". If you guys think this is a bad idea, we should talk. /cc: @manasdk